ONSdigital/response-operations-ui
# Only display trading as name if present

# Motivation and Context
Only display the Trading as name on the RU page if there is one.  Currently the 'Trading as' heading appears below the RU ref with nothing next to it if no trading as name exists.  

The RU name field will still be displayed within each survey for an RU.  This is to avoid a horrible amount of unnecessary logic in the page if a trading as name is added or removed in collection exercises.

# What has changed
If no trading as name is present for the RU, the field will not be shown below the RU name at the top of the page.  Added RU_DETAILS id to the page to help check that the trading as name isn't displayed on the page when running the acceptance tests.

# How to test?
- Run against branch `display-trading-as-name-on-ru-details-only-if-present` in acceptance tests
- Open up the details for any RU (apart from Red Bricks).  The trading as name should be displayed
- Open up the RU details for Red Bricks.  As this doesn't have a trading as name, the field shouldn't be displayed.
- All other tests should pass

# Links
https://github.com/ONSdigital/rasrm-acceptance-tests/tree/display-trading-as-name-on-ru-details-only-if-present
